# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
float64 depth_file_name_number
uint8[] image
float32[] x
float32[] y
float32[] z
uint8[] c
